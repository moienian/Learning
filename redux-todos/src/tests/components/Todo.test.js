import React from "react";
import { shallow } from "enzyme";
import Todo from "../../components/Todo";

describe("<Todo />", () => {
  test("should render <Todo /> correctly", () => {
    const wrapper = shallow(<Todo />);
    expect(wrapper).toMatchSnapshot();
  });
});
