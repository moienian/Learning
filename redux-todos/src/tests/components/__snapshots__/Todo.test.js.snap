// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Todo /> should render <Todo /> correctly 1`] = `<li />`;
